### PR TITLE
translation(js): /Web/JavaScript/Reference/Statements/async_function

### PR DESCRIPTION
**Pull request by AdriandeCita**
Original content: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function

### CODE REVIEW

**Comment by AdriandeCita** [comment]
Оновлено, нові зміни: [2ec681b](https://github.com/mdn/content/commit/2ec681bd9ad77115496f551c62fee2ba50c9007f), [76a015f](https://github.com/mdn/content/commit/76a015ffc44bd29c93e725366ad023171e6c7e7d)

File: files/uk/web/javascript/reference/statements/async_function/index.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+
+Якщо потрібно безпечним чином виконати паралельно два чи більше завдання, слід очікувати в коді на виклик [`Promise.all`](/uk/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) чи [`Promise.allSettled`](/uk/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled).
+
+> **Застереження:** Функції `concurrentStart` та `concurrentPromise` не є функціонально еквівалентними.
+>
+> Якщо у функції `concurrentStart` `швидкий` проміс відхиляється перед сповненням `повільного` промісу, то буде викинуто помилку необробленого відхилення промісу, незалежно від наявності умови `catch` у місці виклику.
+>
+> У функції `concurrentPromise`, `Promise.all` зв'язує ланцюжок промісів за один прохід, тобто операція негайно завершить роботу в разі помилки незалежно від послідовності відхилення промісів, і помилка завжди відбуватиметься всередині налаштованого ланцюжка промісів, що дає змогу її обробити у звичайний спосіб.
```
Comment (undead404) [inline files/uk/web/javascript/reference/statements/async_function/index.md:229]:
```suggestion
> У функції `concurrentPromise` `Promise.all` зв'язує ланцюжок промісів за один прохід, тобто операція негайно завершить роботу в разі помилки незалежно від послідовності відхилення промісів, і помилка завжди відбуватиметься всередині налаштованого ланцюжка промісів, що дає змогу її обробити у звичайний спосіб.
```

File: files/uk/web/javascript/reference/statements/async_function/index.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+    .catch(e => {
+      return downloadFallbackData(url)  // повертає проміс
+    })
+    .then(v => {
+      return processDataInWorker(v)  // повертає проміс
+    })
+}
+```
+
+Цей можна переписати як одну асинхронну функцію, як це показано нижче:
```
Comment (undead404) [inline files/uk/web/javascript/reference/statements/async_function/index.md:247]:
```suggestion
Це можна переписати як одну асинхронну функцію, як це показано нижче:
```

Comment (undead404) [reply files/uk/web/javascript/reference/statements/async_function/index.md:247]: Цейво...

**Review by undead404 — CHANGES_REQUESTED** [review]

**Review by undead404 — APPROVED** [review]

**Pull request merged**